src/routes/contact.js

```diff
@@ -92,6 +93,54 @@ router.post('/', async (req, res) => {
 	return utils.response(res, code, message);
 });
 
+// Sends emails to contacts based on body
+router.post('/send', async (req, res) => {
+	let code;
+	let message;
+	const emails = [];
+
+	try {
+		/** @todo allow for passing entity and contact arrays */
+		const { entityIds, contactIds, relationshipTitle } = req.body;
+		
+		if (entityIds === undefined && contactIds === undefined) {
```

Comment:
It looks like this is only running if `entityIds` and `contactIds` is undefined. If that's the case. are they needed in the request body?

Reply:
That's for future planning. Eventually, we're going to allow for passing an array of either or both types of ID's to _only_ send emails to those associations. For now, though, that functionality isn't in there.